### PR TITLE
feat: add enums related to Minor PP applications and GLA reporting for residential units

### PR DESCRIPTION
**Pull request by jessicamcinchak**
Reflect values that will tracked via new List component schemas introduced here: https://github.com/theopensystemslab/planx-new/tree/main/editor.planx.uk/src/%40planx/components/List/schemas

### CODE REVIEW

**Comment by jessicamcinchak** [comment]
[Add example ODP Schema payload for Minor Planning Permission](https://trello.com/c/XMsbdKCR/2919-add-example-odp-schema-payload-for-minor-planning-permission)

File: types/enums/DevelopmentTypes.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const DevelopmentTypes = {
+  changeOfUse: 'Change of use',
+  changeOfUseFrom: 'Change of use of existing single home',
```
Comment (ianjon3s) [inline types/enums/DevelopmentTypes.ts:3]: Should probably be `Change of use of an existing single home` to be consistent with `Change of use to a home`

File: types/enums/UseClasses.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+export const GLAUseClasses = {
+  bTwo: 'B2 - General industry',
+  bEight: 'B8 - Storage and distribution',
+  cOne: 'C1 - Hotels',
+  cTwo: 'C2 - Residential institutions',
+  cTwoA: 'C2a - Secure residential institutions',
+  cThree: 'C3 - Dwellinghouses',
+  cFour: 'C4 - Houses in multiple occuplation (HMOs)',
```
Comment (ianjon3s) [inline types/enums/UseClasses.ts:8]: Spelling mistake: `occuplation` -> `occupation`

**Review by ianjon3s — APPROVED** [review]
Couple of small spelling/grammar comments

File: types/enums/UseClasses.ts (hunk 2 above)
Comment (jessicamcinchak) [reply types/enums/UseClasses.ts]: Thanks for catching these, both updated!

**Pull request merged**